docs/TOC.md

```diff
@@ -339,6 +339,7 @@
 ### [Known Issues](release-notes/known-issues.md)
 
 ### NuGet 6.x
+#### [NuGet 6.8](release-notes/NuGet-6.8.md)
```

Comment:
@zivkan has a PR about this: https://github.com/NuGet/docs.microsoft.com-nuget/pull/3192.